src/io/flutter/view/InspectorPanel.java

```diff
@@ -82,6 +82,7 @@
   private final InspectorService.FlutterTreeType treeType;
   @NotNull
   private final FlutterApp flutterApp;
+  private final InspectorService inspectorService;
```

Comment:
`@NotNull`, to match the constructor param.

Reply:
done